### ISSUE TITLE
Cart Errors when amount is > 999

### ISSUE THREAD

**Issue by harrygr** [samarkanddesign/samarkand-web#177]
https://sentry.io/samarkand-design/samarkand-prod/issues/320611526/

```
ErrorException: A non well formed numeric value encountered
  File "app/Presenters/Presenter.php", line 32, in handleError
    return money_format(config('shop.currency_symbol').'%i', $value);
  File "app/Presenters/Presenter.php", line 32, in money
    return money_format(config('shop.currency_symbol').'%i', $value);
  File "app/Http/Middleware/RedirectIfCartIsEmpty.php", line 27, in handle
    return $next($request);
...
(47 additional frame(s) were not displayed)

ErrorException: A non well formed numeric value encountered (View: /app/resources/views/shop/cart.blade.php)
  File "app/Presenters/Presenter.php", line 32, in handleViewException
    return money_format(config('shop.currency_symbol').'%i', $value);
  File "app/Http/Middleware/RedirectIfCartIsEmpty.php", line 27, in handle
    return $next($request);
...
(43 additional frame(s) were not displayed)
```

**Comment by harrygr** [comment]
Write a regression test for this

**Comment by harrygr** [comment]
https://sentry.io/samarkand-design/samarkand-staging/issues/320614205/

**Comment by harrygr** [comment]
Fixed by #179
